app/javascript/elm/tests/MainTests.elm

```diff
@@ -345,11 +346,11 @@ crowdMapArea =
                     |> Expect.equal (not onOffValue)
         , fuzz int "UpdateCrowdMapResolution changes the value of model.crowdMapResolution" <|
```

Comment:
You could use `fuzz intRange 1 40` here. https://package.elm-lang.org/packages/elm-community/elm-test/latest/Fuzz

Then you can add a separate test to check the behaviour of UpdateCrowdMapResolution when the value is outside of bounds if you want :)